lib/src/excel.dart

```diff
@@ -48,7 +55,8 @@ class ExcelDart {
   /// Waits for Office APIs to be ready and then creates
   /// a new spreadsheet with [sheetName] and populates the spreadsheet with
   /// entries in the [insertionOrderList].
-  void populate(List<InsertionOrder> insertionOrderList) async {
+  void populate(List<InsertionOrder> insertionOrderList,
+      bool highlightUnderpacing, String threshold) async {
```

Comment:
when would highlighUnderpacing be false?

Reply:
if highlight underpacing is false, does threshold do anything? do these make sense as optional arguments?

Reply:
optional arguments, yes

Reply:
Actually, highlightUnderpacing will still remain as a required parameter, its value will be false if user doesn't check the checkbox. And threshold is no longer a parameter since it is changed to an excel cell input.